packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/edit/OpenstackCredentialsEdit.tsx

```diff
@@ -22,6 +32,29 @@ import {
 export const OpenstackCredentialsEdit: React.FC<EditComponentProps> = ({ secret, onChange }) => {
   const { t } = useForkliftTranslation();
 
+  const insecureSkipVerifyHelperTextMsgs = {
+    error: t('Error: this field must be set to a boolean value.'),
+    successAndSkipped: t("The provider's CA certificate won't be validated."),
+    successAndNotSkipped: t("The provider's CA certificate will be validated."),
+  };
+
+  const cacertHelperTextMsgs = {
+    error: t(
+      'Error: The format of the provided CA Certificate is invalid. Ensure the CA certificate format is valid.',
+    ),
+    success: t(
+      'A CA certificate to be trusted when connecting to the OpenStack Identity (Keystone) endpoint. Ensure the CA certificate format is valid. To use a CA certificate, drag the file to the text box or browse for it. To use the system CA certificates, leave the field empty.',
+    ),
```

Comment:
Is system CA certificates supposed to be plural? [I don't know]

Reply:
both works for me, it's one certificate per our provider, but the system may have more then one for other servers, so both singular and plural are ok with me.

Reply:
I'll change to a singular phrasing since the current text refers to the ca assigned to our provider:
"To use the a system CA certificate, leave the field empty"